### PR TITLE
fix: tooltip component styles

### PR DESCRIPTION
**Pull request by bojangles-m**

### CODE REVIEW

File: .changeset/stupid-poets-bathe.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@frontify/fondue': patch
+---
+
+Fixed tooltip component styles
```
Comment (SamuelAlev) [inline .changeset/stupid-poets-bathe.md:5]:
```suggestion
fix: don't use dark variant for the tooltip styles
```

**Review by SamuelAlev — APPROVED** [review]
Left a small comment

**Review by imoutaharik — COMMENTED** [review]
Perfect! Thanks. Could you just remove the Header from the InContext Story -> 
in-context--with-header-and-footer-and-dark-theme.

As I said, this won't work anymore but no big deal. We should just remove from examples.

**Comment by bojangles-m** [comment]
> Perfect! Thanks. Could you just remove the Header from the InContext Story -> in-context--with-header-and-footer-and-dark-theme.
> 
> As I said, this won't work anymore but no big deal. We should just remove from examples.

Removed

File: .changeset/stupid-poets-bathe.md (hunk 1 above)
Comment (bojangles-m) [reply .changeset/stupid-poets-bathe.md]: Updated

**Review by imoutaharik — APPROVED** [review]

**Pull request merged**